index.js

```diff
@@ -7,12 +10,28 @@ function ask(questionText) {
   });
 }
 
+//----------------async code---------------//
+
 start();
 
 async function start() {
+  let max = 100
+  let min = 1
+  let compNumber = 50
   console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
   let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
   console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
+  let firstGuess = await ask(`Is your number ${compNumber}?`)
+  if (firstGuess === 'yes') {
+    console.log('You got it!')
+  } else if (firstGuess === 'no') {
```

Comment:
For your next step, consider making clear to the user they need to put an H or an L to indicate to the program which path its guessing needs to take. You will also need to assign this await ask to a variable.